frontend/src/metabase/containers/Overworld.jsx

```diff
@@ -269,6 +274,64 @@ class Overworld extends React.Component {
   }
 }
 
+class AdminPinMessage extends React.Component {
+  state = {
+    showMessage: false,
+  };
+
+  static storageKey = "mb-admin-homepage-pin-propaganda-hidden";
+
+  componentWillMount = async () => {
+    const isHidden = await window.localStorage.getItem(this.storageKey);
+    this.setState({
+      showMessage: !isHidden,
+    });
+  };
+  dismissPinMessage = async () => {
+    await window.localStorage.setItem(this.storageKey, true);
```

Comment:
This isn't async. Also I'm pretty sure if you use the `static` keyword above `this.storageKey` is going to be undefined. I'd just make it a `const` at the top of the file.

Reply:
Yeah d'oh. I think I had initially made these functions async cause I thought we might set a backend setting and confused myself.